etc/templates/profile.template

```diff
@@ -221,6 +221,8 @@ include globals.local
 #dbus-user.talk org.freedesktop.Notifications
 #dbus-system none
 
+# Note: read-only entries should usually go on disable-common.inc (especially
```

Comment:
Typo: ... go in disable-common.inc ...

Reply:
> Typo: ... go in disable-common.inc ...

Changed.

By the way, are you sure that there is a preferred form?

I think I've seen it used both ways, so I don't know.

Reply:
> By the way, are you sure that there is a preferred form?

Sorry, I don't understand what preferred form you're referring to in this context.

Reply:
Oh, I just thought it was an actual typo. If you'd like to keep the "on $file" form, please disregard my nitpick.